<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QListWidget" name="SongList">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>201</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>111</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="ControlList">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <item>
       <property name="text">
        <string>Play</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Previous</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Next</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Add Song</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="SoundSize">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Auto_Add">
      <property name="text">
       <string>Scan</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="SongList_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>201</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="listBtn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PlayList</string>
   </property>
  </widget>
  <zorder>SongList_2</zorder>
  <zorder>SongList</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>listBtn</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
